packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -216,7 +216,13 @@ class ObjectForest extends SimpleDependee implements IEditableForest {
         if (path === undefined) {
             return TreeNavigationResult.NotFound;
         }
-        this.moveCursorToPath(path, cursorToMove);
+        // TODO: is this reasonable? This is the case where the path is still "alive", whatever that means
+        // but the node it leads to doesn't actually exist anymore.
```

Comment:
I would have expected `this.anchors.locate` to return `undefined` in that case. Can you say more about why it doesn't? It seems like maybe the forest and the AnchorSet may be out of sync.

Reply:
I had a deeper look, and that's indeed what is happening: the tests need to call `forest.anchors.applyDelta(delta);` to avoid this issue. This is because the forest is not responsible for updating the AnchorSet.